### ISSUE TITLE
Accessibility issue on some primeng components: should be able to set the aria-label 

### ISSUE THREAD

**Issue by meniar-saouat** [primefaces/primeng#10256]
**Current behavior**

In the context of Accessibility ,the problem appears with the **p-calendar** button , to be compliant with "Meaningful Sequence" Rule and for users who rely on screen readers ,  it only reads " button" which is not readable for people who cannot see. Same as "next month" and "previous month" buttons on the date picker. 
The same problem with **p-buttons** and **p-paginator** button , doesn't have support to set aria-label.

 I have this problem when using Lighthouse to test the accessibility of the web application :
- Buttons do not have an accessible name : 

> When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers.

 
**Expected behavior**
I expect to add aria-label to these buttons.

**What is the motivation / use case for changing the behavior?**
We need a solution to conform to WCAG level A.

**Please tell us about your environment:**
Windows 10 Pro , VSCode

* **Angular version:** 
Angular 9 /11

* **PrimeNG version:** 
    primeng: 9/11

* **Browser:**
 All

 
* **Language:**
TypeScript
